cmd/combine/combine.go

```diff
@@ -78,7 +79,9 @@ func Combine(ctx context.Context, inputDir, outputDir string, force bool, opts .
 
 	var combinedKeys []tbls.PrivateKey
 
-	for idx, pkSet := range privkeys {
+	for idx := 0; idx < len(privkeys); idx++ {
```

Comment:
nit: getting confused about different indexes, suggest `valIdx` instead of just `idx`